source/validations.ts

```diff
@@ -17,7 +17,7 @@ class ValidationError extends Error {
 	 * The code must be a string, in snake case and all capital, that starts with
 	 * the substring `ERR_ERL_`.
 	 *
-	 * The message must be a string, starting with a lowercase character,
+	 * The message must be a string, starting with an uppercase character,
 	 * describing the issue in detail.
 	 */
 	constructor(code: string, message: string) {
```

Comment:
Oh, we should update the default message in the `ValidationError` constructor from 
`... See ${url} for more information on this error.` 
to just 
`... See ${url} for more information.`
since it now applies to both warnings and errors.

Reply:
Done